src/templates/components/facilityListing/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { MediaImage } from '@/templates/common/mediaImage'
+import { OperatingStatusFlags } from '@/templates/layouts/healthCareLocalFacility/OperatingStatus'
+import { PhoneNumber } from '@/templates/common/phoneNumber'
+import { formatter as formatParagraphPhoneNumber } from '@/data/queries/phoneNumber'
+import { Address } from '@/templates/layouts/healthCareLocalFacility/Address'
+import { MinimalLocalFacility } from '@/types/formatted/vamcSystem'
+
+type FacilityListingProps = {
+  facility: MinimalLocalFacility
+  /**
+   * The base path for the VAMC system that contains this facility. See the
+   * `OperatingStatusFlags` component's `linkBasePath` prop for more information.
+   */
+  basePath: string
+  /**
+   * I'm not 100% sure this is used anywhere.
+   */
+  type?: 'mobile' | 'desktop'
```

Comment:
I'm not 100% sure it _should_ be. 😅 I don't know what the history of this is in `content-build`, but it _seems_ like it should be handled by responsive or mobile-first design... 🤷